examples/00-mapdl-examples/modal_beam.py

```diff
@@ -0,0 +1,143 @@
+"""
+.. _ref_modal_beam:
+
+=================================
+MAPDL Modal Beam Analysis Example
+=================================
+
+This example demonstrate how to performa a simple modal analysis
+and animate its results.
+
+
+Objective
+~~~~~~~~~
+In this example, we model a simple 3D elastic beam made of BEAM188 elements.
+These beams elements are made of a linear elastic material similar to steel,
+and have a rectangular section.
```

Comment:
```suggestion
This example models a simple 3D elastic beam made of BEAM188 elements.
These beams elements are made of a linear elastic material similar to steel,
and have a rectangular section.
```
Avoid "we". Here I switched to active voice with model as the subject. You could also use "you" in place of "we".